Orchestrator/docs/BFOrchestratorUsage.md

```diff
@@ -18,15 +18,15 @@ The mainstream bot language recognition development cycle with Orchestrator is g
 
 We will use the primary workflow to illustrate how to use Orchestrator commands for full development cycle.
 
-### 1. Prepare label files
+### 1. Create Intent label / utterance examples file in [.lu format][2]
 
 If you are developing a new language model, simply refer to [Language understanding][8] documentation to author label files. Depending on how you plan to design your bot, you may use a single label file corresponding to a single snapshot file for the entire bot or multiple pairs, each for every adaptive dialog where you plan to use Orchestrator as a recognizer. 
 
 In case of migration from legacy dispatch, you may need to retrieve your LUIS application language model using the ```bf luis:version:export``` command to .lu file or QnA Maker knowledgebase using ```bf qnamaker:kb:export```  to .qna format. 
 
 **TBD**: See sample (or example line) here...
 
-### 2. Download Orchestrator base model
+### 2. Download Natural Language Representation ([NLR][4]) base model
```

Comment:
We are no longer calling it NLR, should keep base model

Reply:
Changed to base model.